README.md

```diff
@@ -24,10 +35,25 @@ cdap sandbox start
 
 cdap cli deploy app target/registers-cdap-service-0.0.1-SNAPSHOT.jar
 
-cdap cli start service Sic07.Sic07Service
+cdap cli start service sic07_lookup.Sic07Service
```

Comment:
It appears that this change is to align the service name with `private static final String APP_NAME = "sic07_lookup";` in `Sic07.java`. If that's the case,  I am not sure if it's strictly related to this change (which adds the CH Service). Do you think it's worth pulling this out into a separate commit out of the feature-ch-service branch?

Reply:
@ajorpheus The service name = sic07_lookup  was changed to be able to distinguish the service from the DATASET_NAME as they were initially the same.